public/js/pimcore/document/document.js

```diff
@@ -246,6 +246,19 @@ pimcore.document.document = Class.create(pimcore.element.abstract, {
                 }
 
                 if (this.toolbarButtons.save && this.toolbarButtons.publish) {
+                    if (this.isAllowed("save")) {
+                        const menuItem = this.toolbarButtons.publish.menu.items.items.find(
+                            element => element.text === t('save_draft')
+                        )
+                        menuItem.setHidden(false)
```

Comment:
looks like this condition `!this.data.published` could be tweaked instead 🤔 

https://github.com/pimcore/admin-ui-classic-bundle/blob/cf412952e205a9b0a6000bf61fdfdea47b1b8bab/public/js/pimcore/document/page_snippet.js#L265

is it affected by https://github.com/pimcore/admin-ui-classic-bundle/blob/12309d931e100b2c8f91a0986759e94e12f8a269/public/js/pimcore/document/document.js#L237 ?

Reply:
ahh, nevermind, it's only happening "on the fly" after publishing